src/features/smartSearch/components/filters/Random/index.tsx

```diff
@@ -105,6 +105,9 @@ const Random = ({
                 <MenuItem key={OPERATION.SUB} value={OPERATION.SUB}>
                   <Msg id={localMessageIds.addLimitRemoveSelect.sub} />
                 </MenuItem>
+                <MenuItem key={OPERATION.LIMIT} value={OPERATION.LIMIT}>
+                  <Msg id={localMessageIds.addLimitRemoveSelect.limit} />
```

Comment:
Why not use `messageIds.operators.limit` here as well?

Reply:
Just to clarify, I realize that would break the pattern above, but maybe this is a good time to improve the code that's there and leave it better than we found it?